.github/MAINTAINING.md

```diff
@@ -0,0 +1,88 @@
+# Maintaining the Terraform AWS Provider
+
+<!-- TOC depthFrom:2 -->
+
+- [Pull Requests](#pull-requests)
+    - [Pull Request Review Process](#pull-request-review-process)
+        - [Dependency Updates](#dependency-updates)
+            - [AWS Go SDK Updates](#aws-go-sdk-updates)
+            - [Terraform Updates](#terraform-updates)
+    - [Pull Request Merge Process](#pull-request-merge-process)
+- [Release Process](#release-process)
+
+<!-- /TOC -->
+
+## Pull Requests
+
+### Pull Request Review Process
+
+Notes for each type of pull request are (or will be) available in subsections below.
+
+- If you are planning on owning the pull request through merge/closure, assign it to yourself
```

Comment:
By "through" do you mean the duration (until it's merged/closed) or the action or doing so?

Reply:
The intention here was taking leadership of the pull request until its merged/closed or reassigned. More importantly, a protection mechanism against multiple people working on the same pull request.

Reply:
Makes sense! What about something like "If you plan to be responsible for the pull request, through to the merge/close process ..."?

Reply:
Sounds good. 👍